lib/routes/create_invoice/create_invoice_page.dart

```diff
@@ -74,23 +93,51 @@ class CreateInvoicePageState extends State<CreateInvoicePage> {
   @override
   Widget build(BuildContext context) {
     final texts = context.texts();
+    final themeData = Theme.of(context);
 
     return Scaffold(
       key: _scaffoldKey,
       appBar: AppBar(
         leading: const back_button.BackButton(),
         title: Text(texts.invoice_title),
       ),
-      body: Form(
-        key: _formKey,
-        child: Padding(
-          padding: const EdgeInsets.fromLTRB(16.0, 0, 16.0, 40.0),
-          child: Scrollbar(
-            child: SingleChildScrollView(
-              child: Column(
-                mainAxisSize: MainAxisSize.max,
-                crossAxisAlignment: CrossAxisAlignment.start,
-                children: [
+      body: FutureBuilder<LightningPaymentLimitsResponse>(
```

Comment:
Perhaps a stream builder is more appropriate here instead of maintaining a future?

Reply:
Yes, but this approach is more reliable with apps current state.